demo/estimate_smplx.py

```diff
@@ -0,0 +1,295 @@
+import os
```

Comment:
To help users to get started quickly, shall we provide a short video like [mmpose demo](https://github.com/open-mmlab/mmpose/blob/master/demo/resources/demo.mp4)?

Reply:
We could use `single_person_demo.mp4` in  `demo/resources/`

Reply:
Doesn't seem to be committed and pushed?

Reply:
It has been committed and pushed, the video is [this](https://github.com/open-mmlab/mmhuman3d/blob/main/demo/resources/single_person_demo.mp4).